app/views/fields/has_many/_show.html.erb

```diff
@@ -0,0 +1,38 @@
+<%#
```

Comment:
What exactly is different in this file to the original? I cannot find something...